doc/howto/usage/k8s/k8s_aws_en.md

```diff
@@ -241,21 +249,25 @@ Paste into following inline policies:
     ]
 }
 ```
-
+`Version` : Its value has to be exactly "2012-10-17".
 `AWS_ACCOUNT_ID`: You can get it from following command line:
 
 ```
 aws sts get-caller-identity --output text --query Account
 ```
 
-`MY_CLUSTER_NAME`: Pick a MY_CLUSTER_NAME that you like, you will use it later as well.
+`MY_CLUSTER_NAME`: Pick a MY_CLUSTER_NAME that you like, you will use it later as well. 
+Please note, stack name must satisfy regular expression pattern: [a-zA-Z][-a-zA-Z0-9*]*, which means no "_" or "-" in stack name, or kube-aws will throw error in later steps.
 
 #### External DNS name
 
 When the cluster is created, the controller will expose the TLS-secured API on a DNS name.
 
 DNS name should have a CNAME points to cluster DNS name or an A record points to the cluster IP address.
 
+
+We will need to use DNS name later in tutorial. If you don't already own one, you can choose any DNS name (e.g., `paddle`) and modify `/etc/hosts` to associate cluster ip with that DNS name. We will find the ip to map to `paddle` in later steps. Also in this case, will have to add name service (route53) in aws cluster in later step.
```

Comment:
yup, this is will prevent user from getting confused by local routing and vpc routing, will update in next pull